functions/tslint.json

```diff
@@ -111,7 +111,7 @@
     "prefer-const": {"severity": "warning"},
 
     // Multi-line object literals and function calls should have a trailing comma. This helps avoid merge conflicts.
```

Comment:
I'm fine with removing the trailing comma, but can we remove the comment saying why it's there?

Reply:
The file was auto generated, so there are automatically comments for every rule. The issue in this case was that Tslint could not parse the config file because of the trailing comma in JSON.

I'm fine removing the comments though, maybe a different PR? not sure if it's important.